skeleton/local-deploy/ai-lab.yaml

```diff
@@ -0,0 +1,27 @@
+version: v1.0
+application:
+  type: language
+  name: AI Application
+  description: AI Application Being Locally Deployed
+  containers:
+    - name: llamacpp-server
```

Comment:
Should this instead be whichever model the user selects in `modelServerLocal`? (I recognize that llamacpp is the only choose-able model right now, but it might be good still to not hardcode it)

Reply:
Yeah I think that makes sense. I think alongside this it should only allow users to select model servers that fit the use case, I believe that's why I limited it to llamacpp for the sample as thats what codegen/chatbot need and something like whispercpp might not be a good fit